### DISCUSSION TITLE
NFTS格式的西数硬盘貌似无法休眠

### DISCUSSION THREAD

**Discussion by Leeli0110** [coolsnowwolf/lede#9620] [Bugs / 问题反馈]
西数的mybook 6T 3.5寸的外置硬盘，格式为NTFS，设置休眠貌似不起作用。不知为何原因。
安装hdparm，使用命令hdparm -C /dev/sda，显示bad/missing sense data. drive state is: unknow

**Comment by uniessey** [comment]
我用的外置硬盘与你相同，但主设备上用的是luci-app-hd-idle。这个似乎可以起作用。
